Nodejs/Product/Nodejs/Nodejs.csproj

```diff
@@ -875,6 +875,10 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
+    <ZipItem Include="Templates\Files\TapeUnitTest\UnitTest.js" />
+    <ZipItem Include="Templates\Files\TapeUnitTest\UnitTest.vstemplate" />
+    <ZipItem Include="Templates\Files\TypeScriptTapeUnitTest\UnitTest.vstemplate" />
+    <ZipItem Include="Templates\Files\TypeScriptTapeUnitTest\UnitTest.ts" />
```

Comment:
One minor note which can be addressed later... we'll need to update the VS "15" template manifest to account for this change.
https://github.com/Microsoft/nodejstools/blob/master/Nodejs/Product/Nodejs/Templates/VS_Nodejs.item.vstman